src/qrcode/QRTabBar.tsx

```diff
@@ -5,17 +5,28 @@ import Animated from 'react-native-reanimated'
 import { SafeAreaView } from 'react-native-safe-area-context'
 import { useDispatch } from 'react-redux'
 import SegmentedControl from 'src/components/SegmentedControl'
+import BackChevron from 'src/icons/BackChevron'
 import Share from 'src/icons/Share'
 import Times from 'src/icons/Times'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
 import { TopBarIconButton } from 'src/navigator/TopBarButton'
 import { shareQRCode, SVG } from 'src/send/actions'
 import colors from 'src/styles/colors'
 
 type Props = MaterialTopTabBarProps & {
   qrSvgRef: React.MutableRefObject<SVG>
+  showBackButton: boolean
```

Comment:
Instead of a boolean, can we use an enum? it's not obvious that setting this to false means `X` is used

Reply:
Sure thing, changed it to `qrCloseButton`

Reply:
Could we name this more generic like `closeIcon` / `leftNavIcon`?  Also, can you also make a similar change to the Send / Request screen? Sorry, I should've been more explicit on my original comment.

Reply:
No worries, updated the code so it has a `closeIcon` param on those screens